cosmo_tester/framework/util.py

```diff
@@ -219,12 +221,13 @@ def get_auth_header(username=None, password=None, token=None):
 def create_rest_client(manager_ip,
                        manager_username=None,
                        manager_password=None):
+    headers = get_auth_header(
```

Comment:
Yes but now tenant in auth is mandatory isn't it?
What about community edition?